Examples/FinanceOverview/README.md

```diff
@@ -1,13 +1,24 @@
-# Finance Overview Example Project 
+![Platform](https://img.shields.io/badge/platform-iOS-orange.svg)
+![Languages](https://img.shields.io/badge/languages-swift-orange.svg)
+
+# Finance Overview Example 
 
 This example project shows how to use Tink Money Manager SDK on iOS. The project demonstrates how to configure and display the `FinanceOverviewViewController` within a `UITabBarController`. 
 
 ## Prerequisites
 
-Before you can use the SDK, you need to create a developer account at [Tink Console](https://console.tink.com/). You will also need to have a working integration with Tink to authenticate and authorize users. 
+1. Before you can use the SDK, you need to create a developer account at [Tink Console](https://console.tink.com/).
+2. Follow the [getting started guide](https://docs.tink.com/resources/getting-started/set-up-your-account) to retrieve your `client_id`.
+3. You will also need to have a working integration with Tink to authenticate and authorize users.
 
-## Getting started
+## Configuration
 
 Open `AppDelegate.swift` and replace `YOUR_CLIENT_ID` and `YOUR_ACCESS_TOKEN` with your client id and user access token respectively.
 
-You should now be able to run the test project and try it out.
+If you don't have an access token already, please follow our [guide](https://docs.tink.com/resources/getting-started/get-access-token) on how to generate a new API token. Note that these can expire, so make sure that you're using one that's currently active.
```

Comment:
I would maybe phrase this differently to really point out that it is necessary in the beginning. Maybe something like: 
`Make sure you have a valid access token, if not, you can follow our guide...` wdyt?

Reply:
Good suggestion 👍